mycli/sqlexecute.py

```diff
@@ -354,3 +358,40 @@ def reset_connection_id(self):
     def change_db(self, db):
         self.conn.select_db(db)
         self.dbname = db
+
+    def _create_ssl_ctx(self, sslp):
+        import ssl
+
+        ca = sslp.get("ca")
+        capath = sslp.get("capath")
+        hasnoca = ca is None and capath is None
+        ctx = ssl.create_default_context(cafile=ca, capath=capath)
```

Comment:
If the values for cafile or capath are None, is it still safe to pass them into this function?

Reply:
Yes, they are optional and default to `None`: https://docs.python.org/3/library/ssl.html#ssl.create_default_context